test/unit-tests/compoundPricer.test.ts

```diff
@@ -125,15 +125,11 @@ contract('CompoundPricer', ([owner, random]) => {
     })
     it('should return the new price after resetting answer in underlying pricer', async () => {
       const newPrice = createScaledNumber(1.1)
-      await usdcPricer.setPrice(newPrice)
+      await oracle.setStablePrice(usdc.address, newPrice)
       const cTokenPrice = await cusdcPricer.getPrice()
       const expectedResult = await underlyingPriceToCtokenPrice(new BigNumber(newPrice), exchangeRate, usdc)
       assert.equal(cTokenPrice.toString(), expectedResult.toString())
     })
-    it('should revert if price is lower than 0', async () => {
-      await usdcPricer.setPrice('0')
-      await expectRevert(cusdcPricer.getPrice(), 'CompoundPricer: underlying price is 0')
-    })
```

Comment:
Removed this here cause this testing when USDC price = 0?!
In the Oracle mock this will not fail, using the real Oracle contract this will fail with reason of "could not find Pricer"

Reply:
can you change the test to make sure it reverts?

Reply:
don't delete this test!

Reply:
The Compound pricer us e a mock, and the mock does not use a real underlying pricer, changing this will require alot of refactoring I think.
Any missing use cases should be added to the integration tests.

Also this test does not make sense after merging #311 as an asset should have a stable price > 0 or a pricer contract.

cc @antoncoding